misc/brackets

```diff
@@ -0,0 +1 @@
+Subproject commit 963cbcb18da33c96b61009009f6fb11746f1ca11
```

Comment:
I don't mind, but it probably won't be very convenient to maintain changes in both places. The syntax highlight is currently rather primitive, I hope to improve it in the future.